src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/ItemDetails.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField,
+  TextArea,
+  Datepicker,
+  RepeatableField,
+} from '@folio/stripes/components';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  boolAcceptedValuesOptions,
+  getFieldName,
+  getRepeatableAcceptedValuesPath,
+  getSubfieldName,
+  onAdd,
+  onRemove,
+} from '../../utils';
+import { AcceptedValuesField } from '../../../../../components';
+
+export const ItemDetails = ({
+  contributors,
+  productIdentifiers,
+  initialFields,
+  setReferenceTables,
+  okapi,
+}) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Accordion
+      id="item-details"
+      label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.section`} />}
+    >
+      <Row left="xs">
+        <Col xs={12}>
+          <Field
+            component={TextField}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.title`} />}
+            name={getFieldName(18)}
+          />
+        </Col>
+      </Row>
+      <Row left="xs">
+        <Col xs={4}>
+          <Field
+            component={TextArea}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.receivingNote`} />}
+            name={getFieldName(19)}
+          />
+        </Col>
+        <Col xs={4}>
+          <AcceptedValuesField
+            component={TextField}
+            name={getFieldName(20)}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.mustAcknowledgeReceivingNote`} />}
+            optionValue="value"
+            optionLabel="label"
+            wrapperLabel={`${TRANSLATION_ID_PREFIX}.wrapper.acceptedValues`}
+            acceptedValuesList={boolAcceptedValuesOptions(formatMessage)}
+          />
+        </Col>
+      </Row>
+      <Row left="xs">
+        <Col xs={4}>
+          <Field
+            component={Datepicker}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.subscriptionFrom`} />}
+            dateFormat="MM-DD-YYYY"
+            name={getFieldName(21)}
+          />
+        </Col>
+        <Col xs={4}>
+          <Field
+            component={Datepicker}
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.order.itemDetails.field.subscriptionTo`} />}
+            dateFormat="MM-DD-YYYY"
```

Comment:
```suggestion
            dateFormat="YYYY-MM-DD"
```
and add `dateFormat` prop for all the datepickers